portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/rolesandpermissions/cproles/RolesAdmin.testcase

```diff
@@ -108,6 +108,83 @@ definition {
 			value1 = "No roles were found.");
 	}
 
+	@description = "This test automates LPS-141903 and asserts that users are not counted in Roles after deactivating them"
+	@priority = "3"
+	test DeactivatedUsersAreNotCountedInRoles {
+		task ("Given two users assigned to corresponding sites as a Site Content Reviewer") {
+			JSONGroup.addGroup(groupName = "Site1");
+
+			JSONGroup.addGroup(groupName = "Site2");
+
+			JSONUser.addUser(
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			JSONUser.addUser(
+				userEmailAddress = "user2@liferay.com",
+				userFirstName = "user2",
+				userLastName = "user2",
+				userScreenName = "user2");
+
+			JSONGroup.assignUserToGroup(
+				groupName = "Site1",
+				userEmailAddress = "user1@liferay.com");
+
+			JSONGroup.assignGroupRoleToUser(
+				groupName = "Site1",
+				roleTitle = "Site Content Reviewer",
+				userEmailAddress = "user1@liferay.com");
+
+			JSONGroup.assignUserToGroup(
+				groupName = "Site2",
+				userEmailAddress = "user2@liferay.com");
+
+			JSONGroup.assignGroupRoleToUser(
+				groupName = "Site2",
+				roleTitle = "Site Content Reviewer",
+				userEmailAddress = "user2@liferay.com");
+		}
```

Comment:
Since the steps is repeatable, we can using a for loop here, and combine the Given.